profiles/default.yml

```diff
@@ -4,6 +4,8 @@ charts:
       lines:
         - key:  dotty
           label: bootstrapped
+        - key:  dotty-sbt
+          label: with sbt phases
```

Comment:
Is it OK to have a label with spaces?

Reply:
Yes, it's OK, what this is assumed to test?

Reply:
BTW, this will make the default test time from 2h to 3h. May be we only enable it for merges?

Reply:
Yes, only for merge is fine as long as I can request this benchmark on a PR. New profile?

Reply:
Yes, I think we need a profile for master (`master.yml`), and a specific profile for this one (`sbt.yml`), as usually people may not want it.